CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+## 4.3.0
+
+- Implements Client#close method, allows Infinity for "http2SessionIdleTime" parameter
+- Adds missed tests for FQL queries
+- Fixes cross-platform global object
+- Notifies about new package version and shows release notes on install
+- Exposes parseJSON from utils
+- Adds 'ngram' to special cases for snake_case to CamelCase conversion
+- Adds load tests
+- Automatically create jira ticket on github issue
+- Fixes undefined value in let return an empty object
+- Fixes object with space in key to valid json
```

Comment:
I'm guessing that you might mean something more like this:

```suggestion
- Fixes JSON parsing of field names containing spaces
```

If my guess is incorrect, where is the fix? I can take a look and craft a more accurate description.

Reply:
Here is the fix: https://github.com/fauna/faunadb-js/pull/475
You are correct